Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN if [ "${BUILD_ENV}" = "development" ] ; then \
     fi
 
 # Install sfdx
-RUN mkdir sfdx && wget -qO- https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz | tar xJ -C sfdx --strip-components 1 && ./sfdx/install && rm -rf sfdx
+RUN npm install --global sfdx-cli --ignore-scripts
```

Comment:
@davisagli Despite the [docs saying](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_linux):

> On Linux, the only way to install Salesforce CLI is with a TAR file.

This seems to be how the [Docker version](https://github.com/salesforcecli/sfdx-cli/blob/main/dockerfiles/Dockerfile_full#L15-L16) installs SFDX, and it (apparently) worked for us. 🤷